website/static/js/formViewModel.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Maintains the base class for knockoutJS form ViewModels
+ */
+'use strict';
+
+var ko = require('knockout');
+
+var $osf = require('osfHelpers');
+
+
+var ValidationError = function(errorHeader, errorMessage) {
```

Comment:
Provide sensible defaults for the parameters. The user of this class should only be required to pass in a message or array of messages (see previous comment).

Also, the `error` prefix is unnecessary. `messages`, `title`, and `type` should be fine.